### PR TITLE
Add compiling translations on build

### PR DESCRIPTION
**Pull request by konradko**

### CODE REVIEW

File: docker/cmd-webserver.sh [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/bin/bash -xe
 
 python /usr/src/app/manage.py collectstatic --noinput
+python /usr/src/app/manage.py compilemessages
```
Comment (richtier) [inline docker/cmd-webserver.sh]: this can take a long time to execute (like 60 seconds). can this result in some kind of timeout?

File: makefile [hunk 2]
```diff
@@ -7,13 +7,13 @@ clean:
...
 
-API_CLIENT_ENV_VARS := API_SIGNATURE_SECRET=debug API_CLIENT_BASE_URL=http://debug
 FLAKE8 := flake8 . --exclude=migrations,.venv
 PYTEST := pytest . --cov=. --cov-config=.coveragerc --capture=no $(pytest_args)
 COLLECT_STATIC := python manage.py collectstatic --noinput
+COMPILE_TRANSLATIONS := python manage.py compilemessages
 
 test:
-	$(COLLECT_STATIC) && $(FLAKE8) && $(API_CLIENT_ENV_VARS) $(PYTEST)
+	$(COLLECT_STATIC) && $(COMPILE_TRANSLATIONS) && $(FLAKE8) && $(PYTEST)
```
Comment (richtier) [inline makefile]: I thin the purpose of this is to detect if the translations have a syntax error? Does `makemessages` result in a non-0 if syntax errors occur? If not then the tests will not detect the problem.

File: docker/cmd-webserver.sh (hunk 1 above)
Comment (richtier) [inline docker/cmd-webserver.sh]: oh never mind. we're still expecting the devs to run `makemessages` before committing :+1:

File: makefile (hunk 2 above)
Comment (konradko) [inline makefile]: both fail if there is a syntax error in present `.po` files, .e.g.

```
CommandError: Execution of msgfmt failed: /workspace/dit/directory-ui-supplier/locale/zh_Hans/LC_MESSAGES/django.po:94: keyword "asdasdsad" unknown
/workspace/dit/directory-ui-supplier/locale/zh_Hans/LC_MESSAGES/django.po:94:1: syntax error
msgfmt: found 2 fatal errors
```

**Pull request merged**